docs/public/RELEASE.md

```diff
@@ -0,0 +1,49 @@
+# Release v2024.1 (2024-07-01)
+
+### [Release information]
+ - Translations have not been fully updated in this release. They will be updated in an upcoming release.
+
+### [Breaking changes]
+ - Removes deprecated profile property in Zonemaster-Engine.
+ - Removes deprecated command line option in Zonemaster-CLI.
+ - Requires database update in Zonemaster-Backend.
+
+### [Features]
+ - Updates implementation of test case Basic01 in Zonemaster-Engine.
+ - Makes IDN names being displayed as both A-label and U-label in Zonemaster-GUI.
+
+### [Zonemaster product]
+This version of Zonemaster also consists of the following components. For each component, see its Changes file or Github release notes. This fix version does not change anything in other components. See previous releases for recent changes.
+
+Component            | Github release notes   | Changes file               | Updated in this release
+---------------------|:----------------------:|----------------------------|:----------------------:
+[Zonemaster-LDNS]    | [v4.0.2][ldns-tag]     | [Changes][ldns-Changes]    | Yes
+[Zonemaster-Engine]  | [v6.0.0][engine-tag]   | [Changes][engine-Changes]  | Yes
+[Zonemaster-CLI]     | [v7.0.0][cli-tag]      | [Changes][cli-Changes]     | Yes
+[Zonemaster-Backend] | [v11.2.0][backend-tag] | [Changes][backend-Changes] | Yes
+[Zonemaster-GUI]     | [v4.3.0][gui-tag]      | [Changes][gui-Changes]     | Yes
+
+For more information on previous versions of the Zonemaster product see the [Changes][zonemaster-Changes] file or the [releases] page. For general information ses the [README] file.
+
+[README]: https://github.com/zonemaster/zonemaster/blob/master/README.md
+[releases]: https://github.com/zonemaster/zonemaster/releases
+
+[ldns-tag]: https://github.com/zonemaster/zonemaster-ldns/releases/tag/v4.0.2
+[engine-tag]: https://github.com/zonemaster/zonemaster-engine/releases/tag/v6.0.0
+[cli-tag]: https://github.com/zonemaster/zonemaster-cli/releases/tag/v7.0.0
+[backend-tag]: https://github.com/zonemaster/zonemaster-backend/releases/tag/v11.2.0
+[gui-tag]: https://github.com/zonemaster/zonemaster-gui/releases/tag/v4.3.0
+
+[zonemaster-Changes]: https://github.com/dotse/zonemaster/blob/master/Changes
```

Comment:
```suggestion
[zonemaster-Changes]: https://github.com/zonemaster/zonemaster/blob/master/Changes
```

Reply:
How did that old reference to "dotse" survice?

Reply:
Beats me. But it redirects to `zonemaster/zonemaster` just fine.

Reply:
I guess that is why it has not been detected. The referral will be kept by Github until "dotse" is removed or a repository of the same name is created under "dotse".